src/Core/maps/src/Handlers/Map/MapHandler.Android.cs

```diff
@@ -406,6 +412,13 @@ void OnInfoWindowClick(object? sender, GoogleMap.InfoWindowClickEventArgs e)
 		void OnMapClick(object? sender, GoogleMap.MapClickEventArgs e) =>
 			VirtualView.Clicked(new Devices.Sensors.Location(e.Point.Latitude, e.Point.Longitude));
 
+		string? GetMapId()
```

Comment:
This could be a virtual method people could override, but I guess that could be on net9.0 for example

Reply:
I chose not to make the method virtual because the `CreatePlatformView()` method is already virtual. If the user decides to use a custom `MapHandler` and define their own map options, it will be easier for them to override `CreatePlatformView()` and implement their own creation logic. Additionally, when I was trying the Cloud Maps in MAUI, the first thing I tried to do was add a resource in `Platforms/Android/Resources/values/strings.xml` according to the official [documentation](https://developers.google.com/maps/documentation/get-map-id#add-a-map-id-to-your-app).